easybuild/easyconfigs/b/Boost/Boost-1.65.0-CrayGNU-17.08-python3.eb

```diff
@@ -13,15 +13,21 @@ toolchainopts = {'pic': True, 'usempi': True, 'verbose': False}
 sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
 source_urls = ['https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s/%(version)s']
 
-configopts  = ' --with-python=$EBROOTCRAYPYTHON/bin/python3 --with-python-version=3.5'
-configopts += ' --with-python-root=$EBROOTCRAYPYTHON/lib/python3.5'
+configopts  = ' --with-python=/opt/python/17.06.1/bin/python3'
```

Comment:
patch looks good. 
Can you use 
`--with-python=$EBROOTCRAYPYTHON/bin/python3` ?

Reply:
Yeap no problem